portal-parent/pom.xml

```diff
@@ -241,13 +242,25 @@
                 <version>${hazelcast.version}</version>
             </dependency>
 
-
             <dependency>
                 <groupId>javax.validation</groupId>
                 <artifactId>validation-api</artifactId>
                 <version>${validation-api.version}</version>
             </dependency>
 
+            <!-- Remaining dependency on Markdown4l -->
+            <dependency>
+                <groupId>org.commonjava.googlecode.markdown4j</groupId>
+                <artifactId>markdown4j</artifactId>
+                <version>${markdown4j.version}</version>
+            </dependency>
+
+            <dependency>
```

Comment:
Up to you, but i think that transitive dependency is not a good practice. We use this lib but we not control his version … is not a good idea for me.